templates/opensshd.conf.j2

```diff
@@ -129,6 +129,9 @@ HostbasedAuthentication no
 
 # Enable PAM to enforce system wide rules
 UsePAM {{ 'yes' if ssh_use_pam else 'no' }}
+{% if ssh_google_auth %}
+AuthenticationMethods publickey,keyboard-interactive
```

Comment:
Can you add a comment before this line indicating that it's here because of 2fa?